apis/projectcontour/v1alpha1/accesslog.go

```diff
@@ -251,8 +251,10 @@ func (a AccessLogLevel) Validate() error {
 const (
 	// Log all requests. This is the default.
 	LogLevelInfo AccessLogLevel = "info"
-	// Log only requests that result in an error.
+	// Log only requests that result in an non-success response code.
 	LogLevelError AccessLogLevel = "error"
+	// Log only requests that result in an server error response code..
+	LogLevelServerError AccessLogLevel = "critical"
```

Comment:
The dual naming `critical` vs `LogLevelServerError` feels bit confusing to me. I wonder, would it be bit simpler to just use `LogLevelCritical`, even internally in the code, and just document that critical means server error responses (>500), like you have donoe.